libs/ui/src/hooks/useSmartcard.ts

```diff
@@ -29,32 +28,28 @@ export interface Smartcard {
   writeLongValue(value: unknown | Uint8Array): Promise<Result<void, Error>>;
 }
 
-export type UseSmartcardResult = [
-  smartcard: Smartcard | undefined,
-  hasCardReader: boolean
-];
+export type UseSmartcardResult = [smartcard: Smartcard | undefined];
```

Comment:
If it's just returning the one thing, unwrap the tuple?

```suggestion
export type UseSmartcardResult = Smartcard | undefined;
```